commands/tournaments.js

```diff
@@ -24,4 +24,4 @@ function tournamentsCommand(msg, args) {
 	})
 }
 
-module.exports = tournamentsCommand;
+module.exports = tournamentsCommand;
```

Comment:
No need to commit a file where the only change is the removal of a newline at the end of the file.